src/features/unified-share-modal/SharedLinkSection.js

```diff
@@ -197,6 +193,20 @@ class SharedLinkSection extends React.Component<Props, State> {
                     onCopyError();
                 });
         }
+
+        // if ESL ftux tooltip is showing on initial mount, we call onShow
+        const allowedPermissionLevels = this.getAllowedPermissionLevels();
+
+        if (
+            allowedPermissionLevels.includes(CAN_EDIT) &&
+            sharedLinkEditTooltipTargetingApi &&
+            sharedLinkEditTooltipTargetingApi.canShow &&
+            !isFtuxTooltipShown
```

Comment:
We want to make sure that `onShow` is only triggered once when the tooltip ftux is shown for the first time.